### PR TITLE
Remove the not helpful `unknown` properties from the schema

### PR DESCRIPTION
**Pull request by connerleverett**
Remove some of the the unhelpful `unknown` properties from the schema (specifically Web/Mobile -> API -> Commands and Operations).
Change some `See @link someProperty` to the actual description from the property (specifically Web/Mobile -> API -> Commands and Operations).
Organize Components / Services to come before items referenced by them

### CODE REVIEW

**Comment by rcooney** [comment]
Not sure if this is expected, but there are 227 instances of `See {@link...` in the built output of "Components and Services" now compared to 145 before. Are these just new content or are we failing to create links? It is a little awkward to compare before and after since the sorting changed.

**Comment by connerleverett** [comment]
> Not sure if this is expected, but there are 227 instances of `See {@link...` in the built output of "Components and Services" now compared to 145 before. Are these just new content or are we failing to create links? It is a little awkward to compare before and after since the sorting changed.

I think you have it backwards: 
My branch 
image

Current Dev Center:
image

**Comment by rcooney** [comment]
Haha. I absolutely do.

File: src/components/ViewerMessaging/MessagingDefinitionsSummary.tsx [hunk 1]
```diff
@@ -27,7 +27,24 @@ export default function MessagingDefinitionsSummary(
     return (
         <div>
             {Object.keys(filteredDefinitions)
-                .sort((a, b) => a.localeCompare(b))
+                // Rudimentary sort of components / services first and then the
+                // properties should all come after TODO: Better sort so
+                // components are in one section and services are in another
+                .sort((a, b) => {
+                    if (
+                        a.toLocaleLowerCase().includes("modelproperties") &&
```
Comment (rcooney) [inline src/components/ViewerMessaging/MessagingDefinitionsSummary.tsx:35]: For performance we could do this once at the start
```
const aHasModelProps = a.toLocaleLowerCase().includes("modelproperties");
const bHasModelProps = b.toLocaleLowerCase().includes("modelproperties");
```

Comment (connerleverett) [reply src/components/ViewerMessaging/MessagingDefinitionsSummary.tsx]: Fixed.

**Review by rcooney — APPROVED** [review]

**Pull request merged**